cgroups/systemd/apply_systemd.go

```diff
@@ -487,6 +487,10 @@ func setKernelMemory(c *configs.Cgroup) error {
 		if err != nil {
 			return err
 		}
+	} else if c.MemorySwappiness == -1 {
```

Comment:
@vmarmol 
Thanks for the review
corrected both error format and the patching problem (KernelMemLimit) that happened while porting from the docker environment to libcontainer